mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
+    })
```

Comment:
Inconsistent coding style. The closing curly brace `}` is followed by a comma `,` and then a closing parenthesis `)` on the same line, which is not consistent with the rest of the codebase. This should be corrected to maintain uniformity in coding style.